src/backup.sh

```diff
@@ -96,3 +43,18 @@ if [ "$ENABLE_METRICS" = true ]; then
 fi
 
 echo "Backup complete."
+
+if [ -n "$BACKUP_KEEP_DAYS" ]; then
```

Comment:
We should maybe give later a deeper thought to this part (perhaps add a dry run option?), as it is more dangerous, even though it's disabled by default.